### ISSUE TITLE
Sound goes too fast in The two MAHOU games. Low ADPCM drums. ( the ADPCM sounds low in Batrider too )

### ISSUE THREAD

**Issue by PepAlacant** [psomashekar/pr4m0d-MISTer-bin#18]
Sound goes too fast in The two MAHOU games. plus low ADPCM drums. 

The ADPCM sounds low in Batrider too and voices are barely audible.

Great work !

**Comment by psomashekar** [comment]
Yep, I know about the fast sound issue. Hopefully I can get it resolved soon. For volume boosting, I might make this a configurable value.

**Comment by PepAlacant** [comment]
With the today's release of Garegga core. The sound of Mahou still fast and not ADPCM at all.

**Comment by PepAlacant** [comment]
This is what I can apport about sound. I think the X68000 version uses the same FM chip.
